src/App.module.css

```diff
@@ -0,0 +1,16 @@
+.btnConfirm {
+  background-color: red;
+  color: white;
+  padding: 12px;
+  margin: 10px 0;
```

Comment:
Dont use margins for this. The button needs to be reusable. The final position of the button in exact place needs to be defined by its 'container'. Usualy with flex or grid properties